.github/workflows/binaries.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           echo "$HOME/bin" >> $GITHUB_PATH
           # Install missing build-essential
           sudo apt-get update
-          sudo apt-get install -y build-essential
+          sudo apt-get install -y build-essential python3
```

Comment:
is this an issue with the runner or the node action? we might wanna pin the version to not randomly having binary builds fail (if possible at all)

Reply:
Yes it would be a limitation of the runner, currently in beta. Not clear yet that it solves the issue.